worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/filter/block/CharFilterBlock.java

```diff
@@ -409,6 +409,11 @@ public BiomeType getBiomeType(int x, int y, int z) {
         return getExtent().getBiomeType(x, y, z);
     }
 
+    @Override
+    public BiomeType getBiome(final BlockVector3 position) {
```

Comment:
Are there potentially other places we need to add the override/load?

Reply:
Haven't seen any other occurrences in implementations, which would need that. I could've potentially implemented that in FilterBlock.java - but that could result in a Stackoverflow, if getBiomeType / getBiome is not properly implemented (due to getBiomeType maybe delegating to getBiome, which defaults to getBiomeType)

Reply:
Okay cool, LGTM